docs/THIRDPARTY

```diff
@@ -17,7 +17,7 @@ Comment:
  information or errors found in these notices.
 
 Files: *
-Copyright: 2011-2015 Dropbox, Inc., 2015-2019 Kandra Labs, Inc., and contributors
+Copyright: 2012–2015 Dropbox, Inc., 2015–2021 Kandra Labs, Inc., and contributors
```

Comment:
I thought there was Zulip, Inc? From https://zulip.com/history/
> Zulip, Inc. was acquired by Dropbox in early 2014

Shouldn't it be 2012-2014 Zulip, Inc, 2014-2015 Dropbox, Inc?

Reply:
I wasn’t there, but I assume the acquisition included a copyright transfer agreement.

Reply:
Correct, the original Zulip, Inc. was merged into Dropbox, Inc.